captum/attr/_core/occlusion.py

```diff
@@ -0,0 +1,336 @@
+                            use zero scalar corresponding to each input tensor.
+                            Default: None
+                target (int, tuple, tensor or list, optional):  Output indices for
+                            which difference is computed (for classification cases,
+                            this is usually the target class).
+                            If the network returns a scalar value per example,
+                            no target index is necessary.
+                            For general 2D outputs, targets can be either:
+
+                            - a single integer or a tensor containing a single
+                                integer, which is applied to all input examples
+
+                            - a list of integers or a 1D tensor, with length matching
+                                the number of examples in inputs (dim 0). Each integer
+                                is applied as the target for the corresponding example.
+
+                            For outputs with > 2 dimensions, targets can be either:
+
+                            - A single tuple, which contains #output_dims - 1
+                                elements. This target index is applied to all examples.
+
+                            - A list of tuples with length equal to the number of
+                                examples in inputs (dim 0), and each tuple containing
+                                #output_dims - 1 elements. Each tuple is applied as the
+                                target for the corresponding example.
+
+                            Default: None
+                additional_forward_args (tuple, optional): If the forward function
+                            requires additional arguments other than the inputs for
+                            which attributions should not be computed, this argument
+                            can be provided. It must be either a single additional
+                            argument of a Tensor or arbitrary (non-tuple) type or a
+                            tuple containing multiple additional arguments including
+                            tensors or any arbitrary python types. These arguments
+                            are provided to forward_func in order following the
+                            arguments in inputs.
+                            For a tensor, the first dimension of the tensor must
+                            correspond to the number of examples. For all other types,
+                            the given argument is used for all forward evaluations.
+                            Note that attributions are not computed with respect
+                            to these arguments.
+                            Default: None
+                ablations_per_eval (int, optional): Allows multiple occlusions
+                            to be processed simultaneously in one call to forward_fn.
```

Comment:
This is a little confusing since by default, each occlusion is a separate forward evaluation, and  only when this is > 1, multiple occlusions are included in the same evaluation 
(essentially 'simultaneous'). In the DataParallel case, the multiple occlusions in one batch are divided among the devices. Will reword this to make it clearer, thanks!

Reply:
I see, makes sense. In that case maybe we could say `multiple occlusions included in one batch` ?

Reply:
Sounds good, will do!